admins/README.md

```diff
@@ -0,0 +1,129 @@
+# Administrators
+
+## Table of Contents
```

Comment:
**NOTE:**
This newline requirement also exists for:
* Block quotes
* Code blocks
* Lists (both ordered and unordered)